module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -808,33 +808,35 @@ public function placeHold($holdDetails)
         } catch (Exception $e) {
             throw new ILSException('hold_date_invalid');
         }
-        // Get status to separate Holds (on checked out items) and Pages on a
-        $status = $this->getStatus($holdDetails['item_id']);
         $requestBody = [
-            'requestType' => $item->status->name == 'Available' ? 'Page' : 'Hold',
-            'requestDate' => date('c'),
-            'requesterId' => $holdDetails['patron']['id'],
-            'requester' => [
-                'firstName' => $holdDetails['patron']['firstname'] ?? '',
-                'lastName' => $holdDetails['patron']['lastname'] ?? ''
-            ],
             'itemId' => $holdDetails['item_id'],
+            'requestType' => $holdDetails['status'] == 'Available'
```

Comment:
Is the 'Available' status hard-coded into FOLIO, or might we need to configure this to recognize different statuses under different configurations of the system?

Reply:
From what I've seen, "Available" is hard-coded into FOLIO.

Reply:
I guess the related question is whether there are any other statuses that might influence the type of request we would want to place. If there's any possibility of variation, it would be easy enough to create a configurable map in the class and/or .ini file to correlate status strings with request types (and have an overall default to use if nothing in the map is matched).